global.json

```diff
@@ -3,7 +3,7 @@
     "Microsoft.Build.Traversal": "1.0.45"
   },
   "sdk": {
-    "version": "3.1.301",
+    "version": "5.0.100",
```

Comment:
Why is this change present?

Reply:
It was made initially because build wasn't working locally for me. But this has been updated in compute-2020-12 branch now